appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockFirebaseDB.java

```diff
@@ -126,22 +126,20 @@ public void onFailure(Throwable caught) {
     AUTH_SVC.getToken(projectName, callback);
   }
 
-  // Commented out. We are just putting FirebaseDB in the "Experimental"
-  // Category for now
-  // /**
-  //  * Called when the component is dropped in the Designer window
-  //  * we give a warning that firebase is still experimental.
-  //  */
-
-  // @Override
-  // public void onCreateFromPalette() {
-  //   if (!warningGiven) {
-  //     warningGiven = true;
-  //     Ode.getInstance().warningDialog("There be Dragons here!", "FirebaseDB is an experimental feature " +
-  //       "which may change in the future or break. Packaged Apps built with this component may not " +
-  //       "function into the indefinite future.", "OK");
-  //   }
-  // }
+  /**
+   * Called when the component is dropped in the Designer window
+   * we give a warning that firebase is still experimental.
+   */
+
+  @Override
+  public void onCreateFromPalette() {
+    if (!warningGiven) {
+      warningGiven = true;
+      Ode.getInstance().warningDialog("Warning!", "FirebaseDB is an experimental feature " +
```

Comment:
depending on how long this is going to stay in the codebase, should it be internationalised?